diff -Naur ./mac80211/rx.c ./mac80211_patched/rx.c
--- ./mac80211/rx.c	2021-02-14 17:32:24.000000000 -0500
+++ ./mac80211_patched/rx.c	2022-03-06 11:03:43.036688375 -0500
@@ -2674,7 +2674,7 @@
 	if (ieee80211_data_to_8023_exthdr(skb, &ethhdr,
 					  rx->sdata->vif.addr,
 					  rx->sdata->vif.type,
-					  data_offset))
+					  data_offset, true))
 		return RX_DROP_UNUSABLE;
 
 	ieee80211_amsdu_to_8023s(skb, &frame_list, dev->dev_addr,
